src/translate-once.js

```diff
@@ -13,7 +13,7 @@
   createDirective = function (attribute) {
     var namedDirective = getNamedDirectiveFromAttribute(attribute);
     angular.module(MODULE_NAME).directive(namedDirective, ['$parse', '$translate',
-    TranslateOnceAttributeDirective.bind(undefined, attribute)]);
+    angular.bind(undefined, TranslateOnceAttributeDirective, attribute)]);
```

Comment:
I feel so silly for not realizing this was supported by the angular library. Thanks for this!!